src/backend/utils/misc/etc/hawq-site.xml

```diff
@@ -149,4 +149,39 @@ under the License.
 		</description>
 	</property>
 	<!-- HAWQ resource enforcement parameters end here. -->
+
+    <!-- HAWQ ACL parameters -->
+    <property>
+        <name>hawq_acl_type</name>
+        <value>standalone</value>
+        <description>HAWQ ACL mode. 
+                     'standalone' means HAWQ does native ACL check;
+                     'ranger' means HAWQ does priviliges check through Ranger.
+        </description>
+    </property>
+
+    <property>
+        <name>hawq_rps_address_host</name>
+        <value>localhost</value>
+        <description>Ranger plugin server address hostname.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
+        </description>
+    </property>
+
+    <property>
+        <name>hawq_rps_address_port</name>
+        <value>8432</value>
+        <description>Ranger plugin server address port number.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
+        </description>
+    </property>
+
+    <property>
+        <name>hawq_rps_address_suffix</name>
+        <value>rps</value>
+        <description>Ranger plugin server suffix of restful service address.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
```

Comment:
We can keep the example here. and see it co-works with the two above parameters.